Backend/DynamicQR.Application/QrCodes/Commands/CreateQrCode/CommandHandler.cs

```diff
@@ -1,25 +1,47 @@
-﻿using DynamicQR.Domain.Interfaces;
+using DynamicQR.Domain.Interfaces;
 using DynamicQR.Domain.Models;
 using MediatR;
+using System.Runtime.CompilerServices;
+using System.Text;
 
+[assembly: InternalsVisibleTo("Application.Tests")]
 namespace DynamicQR.Application.QrCodes.Commands.CreateQrCode;
 
 public class CommandHandler : IRequestHandler<Command, Response>
 {
     private readonly IQrCodeRepositoryService _qrCodeRepositoryService;
     private readonly IQrCodeTargetRepositoryService _qrCodeTargetRepositoryService;
+    private const int QrCodeIdLength = 8;
 
     public CommandHandler(IQrCodeRepositoryService qrCodeRepositoryService, IQrCodeTargetRepositoryService qrCodeTargetRepositoryService)
     {
         _qrCodeRepositoryService = qrCodeRepositoryService ?? throw new ArgumentNullException(nameof(qrCodeRepositoryService));
         _qrCodeTargetRepositoryService = qrCodeTargetRepositoryService ?? throw new ArgumentNullException(nameof(qrCodeTargetRepositoryService));
     }
 
+    internal string GenerateQrCodeId()
+    {
+        const string chars = "abcdefghijklmnopqrstuvwxyz0123456789";
+        Random random = new();
+        var result = new StringBuilder(QrCodeIdLength);
+        for (int i = 0; i < QrCodeIdLength; i++)
+        {
+            result.Append(chars[random.Next(chars.Length)]);
+        }
+        return result.ToString();
+    }
```

Comment:
We zouden ook zoiets kunnen doen:

```
        Guid guid = Guid.NewGuid();

        // Convert GUID to string and get the first part
        string guidString = guid.ToString();
        string firstPart = guidString.Split('-')[0];

        return firstPart;
```

Reply:
Maar dan zijn er veel minder ID's mogelijk.
De huidige methode levert 36^QrCodeIdLength
Met de Guid methode zou je 16^QrCodeIdLength hebben

Lijkt me niet dat we dat willen toch? @jensdenbraber

Reply:
goed punt